### DISCUSSION TITLE
Is this safe to use? (question about statistification)

### DISCUSSION THREAD

**Discussion by katastrophe92** [garywill/BigSearch#8] [Q&A] [answered]
It does the job done but...and I'm not a hater 😂. I'm not happy with sending my data to a company that is required by law to report all that data to the Chinese Communist Party. Should I be 100% sure that nothing is sent when user disables the data send?

**Comment by garywill** [comment, marked as answer]
**Update:**
**No analystics** in Big Search 3.x . 
Thank those users who provided their analystics non-sensitive data .
From last year I've got enough non-sensitive data to know what this product should be like, and I don't need more now.
Analysitics is completely removed in latest Big Search. 
Big Search now is 3.x (updated from 2.x) in 2023 

-----------------------

original answer (2.x obsolete):

If you go to the extension settings, and check the "don't send any statistics" checkbox, nothing will be sent. See the code:
https://github.com/garywill/BigSearch/blob/8ac64a92996c0a1ca32a24cf90635ec1d8b90b89/src/js/mtm.js#L6-L11
As long as you have `noaddonstatistics`=true in your settings, statistics won't be enabled. 
Yes it can be 100% sure ~!



For those users who don't check that box, this is what the analystics server gets:
https://github.com/garywill/BigSearch/blob/8ac64a92996c0a1ca32a24cf90635ec1d8b90b89/src/js/mtm.js#L36-L41


I think analytics is needed for developers to improve their products. Though , **I truely respect users who care about privacy. So I've been providing option in settings to disable analystics, since the first release**.

**Comment by katastrophe92** [comment]
@garywill Fair enough. Thanks for the explanation and sorry if my question was uncomfortable. have a nice day
